src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/GenericDocumentService.java

```diff
@@ -98,6 +100,23 @@ public CaseDocument stampDocument(CaseDocument document,
         return toCaseDocument(stampedDocument);
     }
 
+    public CaseDocument appendApprovedDateToPensionOrderDocument(CaseDocument document,
+                                                                 String authorisationToken,
+                                                                 LocalDate approvalDate,
+                                                                 String caseId) {
+        CaseDocument pdfCaseDocument = convertDocumentIfNotPdfAlready(document, authorisationToken, caseId);
+        log.info("Pdf converation if document is not pdf original {} pdfdocument {} for Case ID: {}",
```

Comment:
It's good to see that you've fixed a spelling mistake on line 93 but you introduced a new one here!